docs/dictionary-and-concepts/dictionary-metering.md

```diff
@@ -12,43 +12,61 @@ The terms listed here are all terms relating to the topics of defining metering.
 | Accounting Point |  A domain under balance responsibility where Energy Supplier change can take place and for which commercial business processes are defined. **Additional information:** This is a type of Metering Point. |
 | Asset Type | The technology used on a specific metering point  |
 | **B** |   |
+| Business Reason Code | Indicates the business context, e.g. E23 = 'Periodic metering' used with time series |
```

Comment:
It's probably fine. But what are your thoughts about the ebIX reference vs the enums we use in the domain? Should/could we skip the ebIX or are we forced to live with both value sets?

Reply:
Make sense to remove the ebix codes. Done.